### ISSUE TITLE
settings object not converting into valid aip due to _text tag

### ISSUE THREAD

**Issue by chriswhong** [chriswhong/reef-light-profiles#4]
When parsing aip files, the `xml-js` package is adding `_text` properties.  

These are handled by the app when rendering charts, but converting back to `aip` file results in an invalid file which can't be parsed by the lights.  

`xml-js` allows has a `textFn` option to override how text is handled.  See this issue: https://github.com/nashwaan/xml-js/issues/53

```
<?xml version='1.0'?>
<ramp>
    <header>
        <version>
            <_text>2</_text>
        </version>
        <checksum>1750899007</checksum>
    </header>
    <colors>
        <deep_red>
            <point>
                <intensity>
                    <_text>0</_text>
                </intensity>
                <time>
                    <_text>0</_text>
                </time>
            </point>
...
```

**Comment by chriswhong** [comment]
Closed by 54eb5472bb6517037d982dd39cbef3a4e2ef7544

**Comment by FernandoReynosoMartinez** [comment]
is resolve? i need help with this example!!!!!
